mmdet/models/losses/dice_loss.py

```diff
@@ -0,0 +1,121 @@
+import torch
+import torch.nn as nn
+
+from ..builder import LOSSES
+from .utils import weight_reduce_loss
+
+
+def dice_loss(pred,
+              target,
```

Comment:
preds, targets?
Use plural form, same for the back

Reply:
Most of losses in mmdet uses pred and target.